src/graphql/generated.ts

```diff
@@ -1505,6 +1505,20 @@ export type CreateTransactionInput = {
   titleValues?: InputMaybe<Scalars['String']>;
 };
 
+/** Input data for creating a unique Colony within the Colony Network. Use this instead of the automatically generated `CreateColonyInput` input type */
+export type CreateUniqueColonyInput = {
+  /** Unique identifier for the Colony. This is the Colony's contract address */
+  colonyAddress: Scalars['ID'];
+  /** User id of creator to associate with further invite codes */
+  initiatorAddress: Scalars['ID'];
+  /** Unique identifier for the Colony's native token (this is its address) */
+  tokenAddress: Scalars['ID'];
+  /** The transaction hash of colony creation transaction */
+  transactionHash: Scalars['String'];
+  /** Type of the Colony (regular or MetaColony) */
+  type?: InputMaybe<ColonyType>;
+};
+
```

Comment:
Hmm, I think this is a leftover from before #2225, you might have to restart your dev and run codegen again.